src/bindings/python/tests/test_graph/test_preprocess.py

```diff
@@ -594,7 +594,7 @@ def test_ngraph_set_layout_by_str_thow_exception(layout):
 
     with pytest.raises(RuntimeError) as e:
```

Comment:
If you want to improve asserts here, I would try something like this to remove `assert`:
```suggestion
    with pytest.raises(RuntimeError, match=r"Layout name is invalid"):
```

Reply:
I'd prefer to keep assert